frontend/src/api/k8s/__tests__/inferenceServices.spec.ts

```diff
@@ -140,7 +140,7 @@ describe('assembleInferenceService', () => {
 
   it('should handle name and display name', async () => {
     const displayName = 'Llama model';
-    const resourceName = 'llama-model';
+    const resourceName = 'my-inference-service-test';
```

Comment:
This test seems weird - the resourceName should be a k8s-safe version of the displayName, so `llama-model` makes sense, but below we shouldn't be passing it as the `servingRuntimeName`. In our calls to `mockInferenceServiceModalData` we should pass `name: displayName, k8sName: resourceName` and for this test we shouldn't need to pass a `servingRuntimeName` at all.

Reply:
Fixed